examples/getting_started/pt/fedavg_script_executor_dp_filter_cifar10.py

```diff
@@ -32,11 +35,11 @@
     job.to(controller, "server")
 
     # Define the initial global model and send to server
-    job.to(Net(), "server")
+    job.to(PTModel(Net()), "server")
 
     for i in range(n_clients):
         executor = ScriptExecutor(task_script_path=train_script, task_script_args="")
-        job.to(executor, f"site-{i}", tasks=["train"], gpu=0)
+        job.to(executor, f"site-{i}", tasks=["train"])
```

Comment:
ExecutorApp and Executor are two very different things. ExecutorApp requires special processing because it represents a client site. It should have been named ClientApp, instead of ExecutorApp.
Similarly, ControllerApp should be named ServerApp.

Reply:
Since these different variations of ExecutorApps and ControllerApps will be hidden behind job patterns, do we even need them then? As the variations such as BasicExecutorApp/DLControllerApp simply add additional components, which could instead be done in the job pattern inheritance.